### DISCUSSION TITLE
Dataset

### DISCUSSION THREAD

**Discussion by edreivaj** [PEA-IA-DS-Proyecto-2/Project-2---Predict_Ratings_based_on_text_statements#5] [General]
La pasada semana hablé con Kenny y me comentó que el dataset que nos había pasado era el que teníamos que utilizar. Le realice una reflexión en cuanto a los datos que contiene ese fichero y las pocas alternativas que dejaba para hacer un trabajo profundo y mas serio. Me comentó que lo hablaría con los profesores para ver si lo ampliaban o no.

Yo soy de la opinión de cumplimentar estos datos con algunos externos para darle mayor fuerza al trabajo que vamos a hacer, pero me gustaría saber vuestra opinión porque Kenny me hizo una reflexión muy acertada y no querría que mi voluntad arrastrase al resto a realizar un trabajo que sea mas complejo de acometer, así que si podéis decirme que opináis al respecto. ¿Cumplimentamos los datos con mas o lo dejamos como están?

Saluditos !!!!

**Comment by edreivaj** [comment]
Por cierto, que leyendo lo que he escrito suena mal, la reflexión de  que sea mas complejo de acometer el trabajo, no es en relación a la capacidad técnica que tenemos para hacerlo sino al conocimiento del sector del que nacen los datos. Quería aclararlo. :)

**Comment by SqueezeU** [comment]
A mi tambien parece un "pobre" dataset para crear al final un Dashboard interactivo que sea mas que simple.
Claro mejor utilizamos que nos dan para no hacer una cosas que al final no seria bienvenido.
Pero no tengo problema si decidimos ampliar el dataset con unos datos mas.
Esperamos que dicen los profes y Kenny o arrancamos ya?

**Reply by edreivaj** [reply]
Si tu tambien estás de acuerdo en ampliar, te diria que tambien se lo digas a Kenny para que tenga en cuenta que no soy el único del equipo que piensa lo mismo y si no responden antes del 31 de diciembre decidimos que hacer.
